doc/build/dts/troubleshooting.rst

```diff
@@ -82,13 +82,13 @@ And if you're trying to **set** that property in a devicetree overlay:
 Look at the preprocessor output
 *******************************
 
-To save preprocessor output when using GCC-based toolchains, add
-``-save-temps=obj`` to the ``EXTRA_CFLAGS`` CMake variable. For example, to
+To save preprocessor output when using GCC-based toolchains, enable the
```

Comment:
Actually this is also supported by clang based compilers.

Reply:
Ok, dropped the note entirely since it seems to be supported by most compilers.